doc/man1/flux-account.rst

```diff
@@ -0,0 +1,121 @@
+.. flux-help-section: flux account
+
+===============
+flux-account(1)
+===============
+
+
+SYNOPSIS
+========
+
+**flux** **account** [*COMMAND*] [OPTIONS]
+
+DESCRIPTION
+===========
+
+.. program:: flux account
+
+:program:`flux account` provides an interface to the SQLite database containing
+information regarding banks, associations, queues, projects, and archived jobs.
+It also provides administrative commands like exporting and populating the DB's
+information to and from ``.csv`` files, updating the database when new versions
+of flux-accounting are released, and more.
+
+DATABASE ADMINISTRATION
+=======================
+
+create-db
+^^^^^^^^^
+
+Create the flux-accounting database.
+
+See :man1:`flux-account-create-db` for more details.
+
+pop-db
+^^^^^^
+
+Populate a flux-accounting database with ``.csv`` files.
+
+See :man1:`flux-account-pop-db` for more details.
+
+export-db
+^^^^^^^^^
+
+Export a flux-accounting database into ``.csv`` files.
+
+See :man1:`flux-account-export-db` for more details.
+
+USER ADMINISTRATION
+===================
+
+view-user
+^^^^^^^^^
+
+add-user
+^^^^^^^^
+
+delete-user
+^^^^^^^^^^^
+
+edit-user
+^^^^^^^^^
+
+BANK ADMINISTRATION
+===================
+
+view-bank
+^^^^^^^^^
+
+add-bank
+^^^^^^^^
+
+delete-bank
+^^^^^^^^^^^
+
+edit-bank
+^^^^^^^^^
+
+list-banks
+^^^^^^^^^^
+
+QUEUE ADMINISTRATION
+====================
+
+view-queue
```

Comment:
Sorry I didn't add a review @cmoussa1. I didn't know if this was WIP or not since a lot of these commands were blank, though its obvious what some of them do.

My only confusion is that (I assume) delete-queue doesn't correspond to a flux queue but rather an accounting construct of queues. Might want to clarify that. And again, so sorry I missed this!

Reply:
No problem at all @wihobbs - these are left blank on purpose until they valid entries in them. I was going to go piece by piece when adding these man pages to avoid a huge PR adding slightly detailed man pages for every command.

You make a great point though about `delete-queue`, so I'll be sure to include this when adding man pages for the `queue-*` commands.